hockeysdk/src/main/java/net/hockeyapp/android/metrics/Sender.java

```diff
@@ -361,54 +338,6 @@ private void logRequest(HttpURLConnection connection, String payload) throws IOE
         }
     }
 
-    /**
-     * Reads the response from a connection.
-     *
-     * @param connection the connection which will read the response
-     * @param builder    a string builder for storing the response
-     */
-    protected void readResponse(HttpURLConnection connection, StringBuilder builder) {
-        String result;
-        StringBuilder buffer = new StringBuilder();
-        InputStream inputStream = null;
-
-        try {
-            inputStream = connection.getErrorStream();
-            if (inputStream == null) {
-                inputStream = connection.getInputStream();
-            }
-
-            if(inputStream != null){
-                BufferedReader br = new BufferedReader(new InputStreamReader(inputStream, "UTF-8"));
-                String inputLine;
-                while ((inputLine = br.readLine()) != null) {
-                    buffer.append(inputLine);
-                }
-                result = buffer.toString();
-            }
-            else {
-                result = connection.getResponseMessage();
-            }
-
-            if(!TextUtils.isEmpty(result)) {
-                HockeyLog.verbose(result);
```

Comment:
We just need to make sure the log is not ambiguous, add a prefix then put message in quotes then remove newlines or something. It would be hard to debug if we cannot look at error log response.